### DISCUSSION TITLE
Mucho éxito a todos👩‍🚀🧑‍🚀

### DISCUSSION THREAD

**Discussion by DiegoHC06** [Launch-X-Latam/MisionFrontEnd#30] [General]
El inicio de está travesía. 🚀Es tomarlo con calma y ser autodidacta.  

Si ya tienes conocimiento o no tienes. Lo importante aquí es aprender y trabajar en equipo. 

!Nunca dudes en preguntar Aunque sea Miles de veces ! 

Saludos y mucho éxito xplorers 🤗

**Comment by OATS74** [comment]
Gracias. Donde puedo preguntar. MS team no hay nadie en la cuenta innovación.mx

**Reply by DiegoHC06** [reply]
tienes que completar el [Onboarding](https://github.com/Launch-X-Latam/OnBoarding), lee cada tutorial, todo está bien explicado una vez hecho todos los pasos y el ultimo te manda al link, pero igual lee las instrucciones. cualquier cosa si aún no puedes, puedes comentar de nuevo.

**Comment by Cain-ARG** [comment]
@OATS74 Completaste el formulario al final del OnBoarding? Ese me llevo a Outlook y a los minutos me envio la invitacion al MS Teams. O sino podes trata de comunicarte por sus medios como las redes sociales.
